demo/vagrant/Vagrantfile

```diff
@@ -69,7 +69,7 @@ nomad -autocomplete-install
 SCRIPT
 
 Vagrant.configure(2) do |config|
-  config.vm.box = "bento/ubuntu-18.04" # 18.04 LTS
+  config.vm.box = "bento/ubuntu-20.04" # 18.04 LTS
```

Comment:
I don't think we actually want to do this upgrade yet. Ubuntu 20.04 has a whole bunch of changes with Canonical's use of `snapd` for services that should definitely not be shipped as snaps (ex. Docker), so there's a whole mess of validation we'll need to do for that.

Reply:
Reverted back to 18.04